.ci/containers/downstream-builder/generate_downstream.sh

```diff
@@ -171,9 +171,16 @@ popd
 pushd $LOCAL_PATH
 
 if [ "$REPO" == "terraform" ]; then
+    # See https://github.com/hashicorp/terraform-provider-google/issues/13864
     make generate
 fi
 
+if [ "$REPO" == "terraform" ] && [ $PR_NUMBER == "7255" ]; then
```

Comment:
Restricting adding copyright headers to only the TPG/TPGB downstreams.

Also adding logic to restrict this action to a specific PR, though not sure if that would need to be a separate PR that follows merging this change?